charts/rancher-logging/rancher-logging/3.9.401+up3.9.4/templates/loggings/kube-audit/logging.yaml

```diff
@@ -0,0 +1,100 @@
+{{- if .Values.additionalLoggingSources.kubeAudit.enabled }}
+{{- $auditLogDir := default "/var/log/kube-audit" .Values.additionalLoggingSources.kubeAudit.logDirectory }}
+apiVersion: logging.banzaicloud.io/v1beta1
+kind: Logging
+metadata:
+  name: {{ .Release.Name }}-kube-audit
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "logging-operator.labels" . | indent 4 }}
+spec:
+  controlNamespace: {{ .Release.Namespace }}
+  {{- if .Values.additionalLoggingSources.kubeAudit.loggingRef }}
+  loggingRef: {{ .Values.additionalLoggingSources.kubeAudit.loggingRef }}
+  {{- end }}
+  disableKubernetesFilter: true
+  fluentbit:
+    image:
+      repository: {{ template "system_default_registry" . }}{{ .Values.images.fluentbit.repository }}
+      tag: {{ .Values.images.fluentbit.tag }}
+    inputTail:
+      Tag: "kube-audit"
+      Path: {{ $auditLogDir }}/audit-log.json
+      Parser: json
+      {{- if .Values.fluentbit.inputTail.Buffer_Chunk_Size }}
+      Buffer_Chunk_Size: {{ .Values.fluentbit.inputTail.Buffer_Chunk_Size }}
+      {{- end }}
+      {{- if .Values.fluentbit.inputTail.Buffer_Max_Size }}
+      Buffer_Max_Size: {{ .Values.fluentbit.inputTail.Buffer_Max_Size }}
+      {{- end }}
+      {{- if .Values.fluentbit.inputTail.Mem_Buf_Limit }}
+      Mem_Buf_Limit: {{ .Values.fluentbit.inputTail.Mem_Buf_Limit }}
+      {{- end }}
+      {{- if .Values.fluentbit.inputTail.Multiline_Flush }}
+      Multiline_Flush: {{ .Values.fluentbit.inputTail.Multiline_Flush }}
+      {{- end }}
+      {{- if .Values.fluentbit.inputTail.Skip_Long_Lines }}
+      Skip_Long_Lines: {{ .Values.fluentbit.inputTail.Skip_Long_Lines }}
+      {{- end }}
+    extraVolumeMounts:
+    - source: {{ $auditLogDir }}
+      destination: {{ $auditLogDir }}
+      readOnly: true
+    {{- if not .Values.disablePvc }}
+    {{- with .Values.fluentbit.bufferStorage }}
+    bufferStorage: {{- toYaml . | nindent 6 }}
+    {{- end }}
+    {{- with .Values.fluentbit.bufferStorageVolume }}
+    bufferStorageVolume: {{- toYaml . | nindent 6 }}
+    {{- end }}
+    {{- end }}
+    {{- if .Values.global.psp.enabled }}
+    security:
+      podSecurityPolicyCreate: true
+      roleBasedAccessControlCreate: true
+    {{- end }}
+    {{- $total_tolerations := concat (.Values.tolerations) (.Values.additionalLoggingSources.kubeAudit.tolerations) }}
+    {{- with $total_tolerations }}
+    tolerations: 
+      {{- toYaml . | nindent 6 }}
+    {{- end }}
+    nodeSelector:
+      {{- include "controlplaneSelector" . | nindent 6 }}
```

Comment:
nit: We've since removed `$total_tolerations` in other Logging CRs and refactored the style for these two fields

Reply:
You can probably just use `concat` wrapped in parens after the `with` keyword